config/api/src/main/java/org/jboss/windup/config/metadata/LabelMetadataBuilder.java

```diff
@@ -0,0 +1,42 @@
+package org.jboss.windup.config.metadata;
+
+public class LabelMetadataBuilder extends AbstractLabelsetMetadata implements LabelProviderMetadata
+{
+    private String origin;
+    private String description;
+    private int priority;
+
+    public LabelMetadataBuilder(String ID, String description) {
+        super(ID);
+        this.description = description;
+        this.priority = Integer.MAX_VALUE;
+    }
+
+    public LabelMetadataBuilder(String ID, int priority, String description) {
+        super(ID);
+        this.priority = priority;
+        this.description = description;
+    }
+
+    @Override
+    public String getOrigin()
+    {
+        return origin == null ? super.getOrigin() : origin;
+    }
+
+    public LabelMetadataBuilder setOrigin(String origin)
+    {
+        this.origin = origin;
+        return this;
```

Comment:
Is the fluent required here? could we just a have a basic setter instead?

Reply:
done